packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -468,6 +464,7 @@ class _TextSelectionHandleOverlay extends StatefulWidget {
     @required this.onSelectionHandleChanged,
     @required this.onSelectionHandleTapped,
     @required this.selectionControls,
+    @required this.vsync,
```

Comment:
rather than taking a vsync, use TickerProviderSingleStateMixin

Reply:
Done. For my own enlightenment, why is this better? I was previously just using the same ticker provider as the existing fade-in animation.